simulator/infra/memory/memory.h

```diff
@@ -49,6 +49,9 @@ class FuncMemory
         template<typename T>
         void write( T value, Addr addr, T mask = all_ones<T>());
 
+        Byte read_byte( Addr addr) const;
```

Comment:
These functions are intentionally non-safe, so they are private. To get access, `read<Byte>` and `write<Byte>` should be used